app/templates/global_var_test.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
There shouldn't be a file used just for testing in the `templates` directory.
Check the documentation for functions that allow you to load templates from strings instead.